opm/simulators/wells/MSWellHelpers.hpp

```diff
@@ -206,7 +206,8 @@ namespace mswellhelpers
     template <typename ValueType>
     ValueType velocityHead(const double area, const ValueType& mass_rate, const ValueType& density)
     {
-        return (0.5 * mass_rate * mass_rate / (area * area * density));
+        const double sign = mass_rate < 0. ? 1.0 : - 1.0;
+        return (sign*0.5 * mass_rate * mass_rate / (area * area * density));
```

Comment:
Are you 100% sure this is correct? I think the head should not depend on the sign here.

If correct, I would prefer to keep this positive, and change the sign in the calling code.

Reply:
For an injector the sign of P_seg - P_outlet vil be different and hence the "losses" needs to change sign. From a formulation point of view it would purchase be better to change the sign for the segment pressure and the outlet segment pressure.
In the current kode sign is used for the "losses" instead. I went for the least intrusive change.

Reply:
The outlet is defined by the distance to the well head. I mean, the definition of the outlet is not associated with flow direction, at least in the code. Then `seg` and `outlet` will refer to the same two segments. Am I missing something here?

Reply:
I think that the sign of (P_seg - P_outlet) will not change depending on flow direction. Typically it will be positive for both injectors and producers for a vertical well, for example.